tools/autograd/gen_autograd.py

```diff
@@ -44,13 +44,15 @@
     'expand': 'self',
     'narrow': 'self',
     'permute': 'self',
+    'reshape_as': 'self',
     'select': 'self',
     'slice': 'self',
     'squeeze': 'self',
     't': 'self',
     'transpose': 'self',
     'unfold': 'self',
     'unsqueeze': 'self',
+    'flatten': 'flatten',
```

Comment:
The value for the key should be the argument that the output is a view of. Here it should be 'self' I think.

Reply:
Ooof. Good catch.